src/mtcp/mtcp_restart.c

```diff
@@ -692,6 +692,23 @@ compute_vdso_vvar_addr(RestoreInfo *rinfo)
   mtcp_sys_close(mapsfd);
 }
 
+NO_OPTIMIZE
+static void
+unmap_one_memory_area(Area *area, int mapsfd)
```

Comment:
Let's declare this function in the beginning along with other internal routines.

Reply:
I agree:
> Let's declare this function in the beginning along with other internal routines.